### PR TITLE
add info about GPUs

### PR DESCRIPTION
**Pull request by crclark96**
compare GPU to CPU in terms of architecture, include computational frameworks used to interface w/ GPUs

### CODE REVIEW

File: src/pages/computer-hardware/gpu/index.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
 ---
 title: Gpu
```
Comment (Bouncey) [inline src/pages/computer-hardware/gpu/index.md]: Capitalise GPU please :+1:

File: src/pages/computer-hardware/gpu/index.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+
+GPUs can be used for computation as well as video rendering. Common ways to do this include OpenACC, CUDA, OpenCL, and OpenGL. Some applications include GPU implementations to reduce the amount of time the application takes to run.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
-
+https://en.wikipedia.org/wiki/Graphics_processing_unit
```
Comment (Bouncey) [inline src/pages/computer-hardware/gpu/index.md]: markdown links please

`[text to click](https://awesome-uri.com)`

**Comment by crclark96** [comment]
changes made

**Comment by Bouncey** [comment]
:sparkles: :sparkles: :sparkles: :sparkles:

**Pull request merged**